packages/@lwc/engine-core/src/framework/renderer.ts

```diff
@@ -65,4 +65,6 @@ export interface RendererAPI {
         connectedCallback?: LifecycleCallback,
         disconnectedCallback?: LifecycleCallback
     ) => E;
+
+    ownerDocument(elm: E): Document;
```

Comment:
```suggestion
    ownerDocument(elm: E): Document | null;
```

It's [possible](https://developer.mozilla.org/en-US/docs/Web/API/Node/ownerDocument) for this to be `null` if called on the document itself.

Reply:
Yes, technically, but this`ownerDocument` will always be called from within a `LightningElement` and we'll never get `null`.

Reply:
OK in that case, it's fine. We don't need the `| null`.